src/test/java/seedu/address/logic/parser/AddClientCommandParserTest.java

```diff
@@ -122,4 +130,36 @@ public void parse_invalidValue_failure() {
                 + ADDRESS_DESC_BOB + PRODUCT_DESC_PRODUCT2 + PRODUCT_DESC_PRODUCT1,
                 String.format(MESSAGE_INVALID_COMMAND_FORMAT, AddClientCommand.MESSAGE_USAGE));
     }
+
+    @Test
+    public void parse_invalidBirthday_failure() {
+        // boundary value analysis
```

Comment:
perhaps you can mention the EP whose boundary you are taking

Reply:
thought the 1 jan and 31 dec etc was clear, but will make it clearer, thanks!